docs/guide/extending_ez_platform_ui.md

```diff
@@ -165,6 +167,250 @@ $menu->reorderChildren(
+{
+    public function process(ContainerBuilder $container)
+    {
+        if (!$container->hasDefinition(TabRegistry::class)) {
+            return;
+        }
+
+        $tabRegistry = $container->getDefinition(TabRegistry::class);
+        $tabGroupDefinition = new Definition(
+            TabGroup::class,  // or any class that extends TabGroup
+            ['custom-tab-group']
+        );
+        $tabRegistry->addMethodCall('addTabGroup', [$tabGroupDefinition]);
+    }
+}
+```
+
+A shorter way that you can use when no custom logic is required, is to add your own tab with the new group.
+If a tab's group is not defined, it will be created automatically.
+
+A new tab group can be rendered using the `ez_platform_tabs` Twig helper:
+
+``` jinja
+<div class="my-tabs">
+    {{ ez_platform_tabs('custom-tab-group') }}
+</div>
+```
+
+This will render the group and all tabs assigned to it.
+
+### Adding a new tab
+
+Before you add a tab to a group you must create the tab's PHP class and define it as a Symfony service with the `ezplatform.tab` tag:
+
+``` yaml
+# services.yml
+
+AppBundle\Custom\Tab:
+    parent: EzSystems\EzPlatformAdminUi\Tab\AbstractTab
+    tags:
+        - { name: ezplatform.tab, group: 'custom-tab-group' }
+```
+
+This configuration also assigns the new tab to `custom-tab-group`.
+
+!!! tip
+
+    If the `custom-tab-group` does not yet exist, it will be created automatically at this point.
+
+The tab class can look like this:
+
+``` php
+// src/AppBundle/Custom/Tab.php
+
+namespace AppBundle\Custom;
+
+use EzSystems\EzPlatformAdminUi\Tab\AbstractTab;
+
+class Tab extends AbstractTab
+{
+    public function getIdentifier(): string
+    {
+        return 'custom-tab';
+    }
+
+    public function getName(): string
+    {
+        return /** @Desc("Custom Tab") */
+            $this->translator->trans('custom.tab.name', [], 'some_translation_domain');
+    }
+
+    public function renderView(array $parameters): string
+    {
+        // Do rendering here
+
+        return $this->twig->render('AppBundle:my_tabs/custom.html.twig', [
+            'foo' => 'Bar!',
+        ]);
+    }
+}
+```
+
+Beyond the `AbstractTab` used in the example above you can use two specializes tab types:
+
+- `AbstractControllerBasedTab` enables you to embed the results of a controller action in the tab.
+- `AbstractRouteBasedTab` embeds the results of the selected routing, passing applicable parameters.
+
+### Modifying tab display
+
+You can order the tabs by making the tab implement `OrderedTabInterface`.
+The order will then depend on the numerical value returned by the `getOrder` method.
+
+``` php
+class Tab extends AbstractTab implements OrderedTabInterface
+{
+    public function getOrder(): int
+    {
+        return 100;
+    }
+}
+```
+
+The tabs will be displayed according to this value in ascending order.
+
+!!! tip
+
+    It is good practice to reserve some distance between these values, for example by multiplying them by 10.
```

Comment:
> by multiplying them by 10.

I suggest to use a formulation with the _step_ notion, it's from 10 to 10 more, like scale bars, not a multiplication. (I don't know if I am clear, here.)

Reply:
Sorry, I totally don't understand what you mean here

Reply:
We want to space the values from 10, it's a step.
item10, item 20, item 30, etc.
So later, we can put an item 21 after item 20 if needed.